docs/migration.md

```diff
@@ -2,96 +2,253 @@
 description: Migrating from Pydantic V1.
 ---
 
-## Install Pydantic V2 alpha
+The Pydantic V2 beta introduces a number of changes to the API, including some breaking changes.
 
-Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in V2.
+We believe Pydantic V2 is ready for your production applications. This page provides a guide highlighting the most
+important changes to help you migrate your code from Pydantic V1 to Pydantic V2.
 
-To get started with the Pydantic V2 alpha, install it from PyPI.
+## Install Pydantic V2 beta
+
+Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in
+Pydantic V2.
+
+To get started with the Pydantic V2 beta, install it from PyPI.
 We recommend using a virtual environment to isolate your testing environment:
 
 ```bash
-pip install --pre -U "pydantic>=2.0a1"
+pip install --pre -U "pydantic>=2.0b1"
 ```
 
-!!! warning "Alpha release!"
-    Note that there are still some rough edges and incomplete features, and while trying out the Pydantic V2 alpha releases you may experience errors.
-
-    We encourage you to try out the alpha releases in a test environment and not in production.
-    Some features are still in development, and we will continue to make changes to the API.
+!!! warning "Beta release!"
+    Note that there are still some rough edges, and while trying out the Pydantic V2 beta releases you may experience errors.
+
+    We encourage you to try out the beta releases in a test environment and not in production.
```

Comment:
In a couple of lines above it says: "We believe Pydantic V2 is ready for your production applications." 🤔